metron-deployment/vagrant/full-dev-platform/README.md

```diff
@@ -62,41 +60,46 @@ Now that the hard part is done, start the Metron deployment process.
   vagrant provision
   ```
 
-  In addition to re-running the entire provisioning play book, you may now re-run an individual Ansible tag or a collection of tags in the following ways.
+### Explore Metron
 
-  ```
-  ./run_ansible_role.sh web
-  ```
-  or
-  ```
-  vagrant --ansible-tags="web" provision
-  ```
-  Will re-run the web role on the Vagrant image. This will re-install (if necessary) and start the UI.
+Navigate to the following resources to explore your newly minted Apache Metron environment.
 
-  A collection of tags is specified as a comma separated list.
+ - [Metron](http://node1:8080)
+ - [Ambari](http://node1:5000)
+ - [Services](http://node1:2812)
 
-  ```
-  ./run_ansbile_role.sh "sensors,enrichment"
+Connecting to the host through SSH is as simple as running the following command.
 
-  ```
+```
+vagrant ssh
+```
 
-  Tags are listed in the playbooks, some frequently used tags:
-  + hdp-install - Install HDP
-  + hdp-deploy - Deploy and Start HDP Services (will start all Hadoop Services)
-  + sensors - Deploy and Start Sensors.
-  + enrichment - Deploy and Start Enrichment Topology.
+### Working with Metron
 
-  Note: there is a convienence script, ```./run_enrichment_role.sh```,  which runs the enrichment tag.
+In addition to re-running the entire provisioning play book, you may now re-run an individual Ansible tag or a collection of tags in the following ways.  The following commands will re-run the `web` role on the Vagrant image. This will install components (if necessary) and start the UI.
 
-### Explore Metron
+```
+./run_ansible_role.sh web
+```
+or
 
-Navigate to the following resources to explore your newly minted Apache Metron environment.
+```
+vagrant --ansible-tags="web" provision
+```
 
- - [Metron](http://node1:8080)
- - [Ambari](http://node1:5000)
+#### Using Tags
 
-Connecting to the host through SSH is as simple as running the following command.
+A collection of tags is specified as a comma separated list.
+
+```
+./run_ansbile_role.sh "sensors,enrichment"
```

Comment:
I don't see a typo.  What am I missing?   The diff may just look weird because I reorganized the content.   [Check out the full rendered version.](https://github.com/nickwallen/incubator-metron/tree/METRON-156-2/metron-deployment/vagrant/full-dev-platform/README.md)